viewing_party/party.py

```diff
@@ -0,0 +1,169 @@
+from collections import Counter
+
+'''
+    wave 1 - question: Did I over use helper function?
+'''
+def create_movie(movie_title, genre, rating):
+    if  genre== None  or  movie_title == None or  rating ==None:
+        return None
+    else:
+        new_movie={
+        "title": movie_title,
+        "genre": genre,
+        "rating": rating}
+        return new_movie
```

Comment:
You can move this code outside the else. In general if indentation isn't necessary, we prefer less indentation